Sources/Views/KMConversationInfoView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  KMConversationInfoView.swift
+//  KommunicateChatUI-iOS-SDK
+//
+//  Created by sathyan elangovan on 16/03/23.
+//
+
+import Foundation
+import UIKit
+
+open class KMConversationInfoView: UIView {
+    private var viewModel: KMConversationInfoViewModel?
+    
+    private let leadingImageView: UIImageView = {
+        let imageView = UIImageView(frame: CGRect.zero)
+        var backImage = UIImage(named: "file")
```

Comment:
Is it mandatory to pass? If yes, then we need to document the same.

Reply:
no need to pass here.removing the lines. it will be fetched from view model